client/components/pages/editor/sections/Hero.jsx

```diff
@@ -27,8 +27,8 @@ export default class Hero extends Component{
   render(){
     return (
       <form>
-        <ShortTextField label="Title" onChange={this.handleChange} value={this.props.data.title} name="title"/>
-        <LongTextField label="Description" onChange={this.handleChange} value={this.props.data.description} name="description"/>
+        <ShortTextField maxLength="60"label="Title" onChange={this.handleChange} value={this.props.data.title} name="title"/>
```

Comment:
I haven't tested it, hence the "Once I can test this in local I will assign someone to review" in the pr description.

You are probably right, I read this from the react docs:
`In React, all DOM properties and attributes (including event handlers) should be camelCased. For example, the HTML attribute tabindex corresponds to the attribute tabIndex in React.`

https://facebook.github.io/react/docs/dom-elements.html

So next step (once I test it) is to find where the actual inputs are, and then add the `maxLength="__"` to that file? I can do that tonight. 🌃